process/templates/PROJECT_NAME_graduation_stage.md

```diff
@@ -0,0 +1,36 @@
+## Project graduation application
+
+### List of project maintainers
+The project must have maintainers with a minimum of three different company affiliations.
```

Comment:
Suggest "organizational" instead of "company" for reasons discussed before.

Reply:
Since PR#117 is landed, should we update and resolve this?